modules/cluster-logging-collector-collector.adoc

```diff
@@ -57,4 +57,3 @@ nodeSpec:
       type: "fluentd" <1>
 ----
 <1> Set the log collector to `fluentd`.
-////
```

Comment:
Are you removing this `////` on purpose?

This file is not currently used. While doing clean-up, would you mind removing the `logging/cluster-logging-collector.adoc` from _Modules included in_?

Reply:
> Are you removing this //// on purpose?

Yeah, no need to keep comments around in adoc ;).  If the defaults need to change at some point, that should be tracked in a bug or RFE or whatever.

> Also, this file is not currently used.

Done with #18284.  Looks like there's some other unused logging content as well, e.g. nobody consuming `modules/cluster-logging-collector-undefined.adoc`, but I've left that alone because I don't really understand how these adoc files are wired together.

Reply:
I hid that whole chunk as at the time the option to switch the logging collector to Rsyslog was still in consideration. It appears that is not the case now. If we remove the `////` at the end, all the material after Line 13 should come out also.